core/src/main/java/bisq/core/network/p2p/inventory/messages/GetInventoryRequest.java

```diff
@@ -21,9 +21,11 @@
 import bisq.common.app.Version;
 import bisq.common.proto.network.NetworkEnvelope;
 
+import lombok.EqualsAndHashCode;
 import lombok.Value;
 
 @Value
```

Comment:
That should also be replaced by a @Getter and @ToString.

Reply:
Commit  c2c1ac2 
Removes `@Value`, `@EqualsAndHashCode(callSuper = false)`
Adds `@Getter`,`@ToString`

I'm not absolutely certain this is what you asked for (by removing `@Value`), but keeping `@Value` , `@Getter`, `@ToString` is redundant because `@Getter`, `@ToString` are implicitly defined by `@Value`.

Reply:
I meant that @Value produces Lombok warnings as well so replace it with EqualsAndHashCode, Getter and ToString